packages/next/src/server/stream-utils/node-web-streams-helper.ts

```diff
@@ -73,6 +73,28 @@ export function streamFromString(str: string): ReadableStream<Uint8Array> {
   })
 }
 
+export function streamFromBuffer(chunk: Buffer): ReadableStream<Uint8Array> {
```

Comment:
Any particular reason you are using Node.js buffers here? They could potentially be missing from Edge runtime environments. 

AFAIK you could just use `Uint8Array` natively and it'll work across platform.

Reply:
Buffer is available in edge runtime https://vercel.com/docs/functions/edge-middleware/edge-runtime#compatible-node.js-modules

Reply:
It is less performant though. I encourage you to read this article: https://sindresorhus.com/blog/goodbye-nodejs-buffer

I pretty much always default to `Uint8Array` now. Other than supporting old Node.js versions/apis that explicitly require a Buffer, Uint8Array is interoperable and much more performant both in size and speed.